### PR TITLE
MODEBSNET-53 skip empty locations processing

### PR DESCRIPTION
**Pull request by grabsefx**
## Purpose
https://issues.folio.org/browse/MODEBSNET-53

## Approach
* skip empty locations processing

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate action.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code are 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by azizbekxm** [comment]
Hello, @grabsefx, Could you please update version to next major version. Spring migration considered as major changes. I mistakenly changed to 1.4.0 -> 1.5.0. but it should be actually 1.4.0 -> 2.0.0.
image

**Comment by azizbekxm** [comment]
Hello, could you please change version to 2.0.0 in your pr.  spring migration considered major changes it should have been 1.4.0 -> 2.0.0. it saves time to create another pr for it.
image

File: src/main/java/org/folio/ebsconet/mapper/OrdersMapper.java [hunk 1]
```diff
@@ -172,18 +186,18 @@ else if ((poLine.getCost().getQuantityElectronic() > 1 && poLine.getCost().getQu
...
+      }
     }
   }
 
   private void redistributeSinglePeMixItem(CompositePoLine poLine) {
     poLine.getCost().setQuantityElectronic(1);
     poLine.getCost().setQuantityPhysical(1);
 
-    // set all location quantities to zero
-    clearLocationQuantities(poLine);
```
Comment (SerhiiNosko) [inline src/main/java/org/folio/ebsconet/mapper/OrdersMapper.java:185]: Is it possible if we have for example 3 locations in composite po line, so now after we deleting clearLocationQuantities() the third one will contain always old values?

Comment (SerhiiNosko) [reply src/main/java/org/folio/ebsconet/mapper/OrdersMapper.java:185]: I see now, this method invoked only from the single place where clearLocationQuantities90 invoked, comment is resolved

**Review by SerhiiNosko — APPROVED** [review]

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**